[js] Windows is not able to find command

### What happened?

`npm` and `pnpm` commands are not found in Windows

### Please provide runtime information.

Windows
NPM and PNPM using Corepack

### Relevant log output

```shell
Error: Exception in thread "main" java.lang.RuntimeException: Unable to find or run pnpm executable 'pnpm'. Please ensure pnpm is installed and available in your PATH.
	at com.redhat.exhort.tools.Operations.getExecutable(Operations.java:284)
	at com.redhat.exhort.providers.JavaScriptProvider.<init>(JavaScriptProvider.java:64)
	at com.redhat.exhort.providers.JavaScriptPnpmProvider.<init>(JavaScriptPnpmProvider.java:33)
	at com.redhat.exhort.providers.JavaScriptProviderFactory.create(JavaScriptProviderFactory.java:37)
	at com.redhat.exhort.tools.Ecosystem.resolveProvider(Ecosystem.java:92)
	at com.redhat.exhort.tools.Ecosystem.getProvider(Ecosystem.java:81)
	at com.redhat.exhort.impl.ExhortApi.componentAnalysis(ExhortApi.java:457)
	at com.redhat.exhort.cli.App.executeComponentAnalysis(App.java:164)
	at com.redhat.exhort.cli.App.executeCommand(App.java:137)
	at com.redhat.exhort.cli.App.main(App.java:56)
Caused by: java.io.IOException: Cannot run program "pnpm": CreateProcess error=2, The system cannot find the file specified
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at com.redhat.exhort.tools.Operations.getExecutable(Operations.java:275)
	... 9 more
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:505)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:158)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
	... 11 more
```

### How can this issue be reproduced?

_No response_

--- a/src/main/java/com/redhat/exhort/providers/JavaScriptNpmProvider.java
+++ b/src/main/java/com/redhat/exhort/providers/JavaScriptNpmProvider.java
@@ -16,6 +16,7 @@
 package com.redhat.exhort.providers;
 
 import com.redhat.exhort.tools.Ecosystem;
+import com.redhat.exhort.tools.Operations;
 import java.nio.file.Path;
 
 /**
@@ -25,7 +26,7 @@ import java.nio.file.Path;
 public final class JavaScriptNpmProvider extends JavaScriptProvider {
 
   public static final String LOCK_FILE = "package-lock.json";
-  public static final String CMD_NAME = "npm";
+  public static final String CMD_NAME = Operations.isWindows() ? "npm.cmd" : "npm";
 
   public JavaScriptNpmProvider(Path manifest) {
     super(manifest, Ecosystem.Type.NPM, CMD_NAME);

--- a/src/main/java/com/redhat/exhort/providers/JavaScriptPnpmProvider.java
+++ b/src/main/java/com/redhat/exhort/providers/JavaScriptPnpmProvider.java
@@ -18,6 +18,7 @@ package com.redhat.exhort.providers;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.redhat.exhort.tools.Ecosystem;
+import com.redhat.exhort.tools.Operations;
 import java.nio.file.Path;
 
 /**
@@ -27,7 +28,7 @@ import java.nio.file.Path;
 public final class JavaScriptPnpmProvider extends JavaScriptProvider {
 
   public static final String LOCK_FILE = "pnpm-lock.yaml";
-  public static final String CMD_NAME = "pnpm";
+  public static final String CMD_NAME = Operations.isWindows() ? "pnpm.cmd" : "pnpm";
 
   public JavaScriptPnpmProvider(Path manifest) {
     super(manifest, Ecosystem.Type.PNPM, CMD_NAME);

--- a/src/main/java/com/redhat/exhort/providers/JavaScriptYarnProvider.java
+++ b/src/main/java/com/redhat/exhort/providers/JavaScriptYarnProvider.java
@@ -33,7 +33,7 @@ import java.util.regex.Pattern;
 public final class JavaScriptYarnProvider extends JavaScriptProvider {
 
   public static final String LOCK_FILE = "yarn.lock";
-  public static final String CMD_NAME = "yarn";
+  public static final String CMD_NAME = Operations.isWindows() ? "yarn.cmd" : "yarn";
 
   private static final Pattern versionPattern = Pattern.compile("^([0-9]+)\\.");
 

